monitor/Dockerfile

```diff
@@ -18,5 +18,6 @@ RUN mkdir -p /etc/artemis/ && \
     mkdir -p /var/log/artemis/
 
 COPY ./configs/* /etc/artemis/
+COPY ./supervisor.d/* /etc/supervisor/conf.d/
```

Comment:
why is this required? Is the default location changed?

Reply:
they weren't default configuration so a volume was necessary. I added them on the image so if someone runs the image by itself without mounting it should work